main/src/library/Debug.flix

```diff
@@ -78,49 +78,49 @@ namespace Debug {
                 }
             case y: Array[Unit, _] =>
                 import static java.util.Arrays.toString(Array[##java.lang.Object, Static]): String;
```

Comment:
@mlutze Could we not import toString as pure??

Reply:
We could but it would be a lie :)

Reply:
But we cast it right after?

Reply:
But ok, point taken.